lib/websocket_server.rb

```diff
@@ -51,6 +57,7 @@ def init_proxy(env, url)
     console = SystemConsole.find_by!(:url_secret => url)
     proxy = WebsocketProxy.new(env, console)
     return proxy.cleanup if proxy.error
+    @logger.info("Starting websocket proxy for VM #{console.vm_id}")
```

Comment:
Maybe make an `attr_accessor :logger` so you just call logger except where you're setting up the logger.